query/src/main/scala/filodb/query/exec/rangefn/AggrOverTimeFunctions.scala

```diff
@@ -222,6 +222,78 @@ class SumAndMaxOverTimeFuncHD(maxColID: Int) extends ChunkedRangeFunction[Transi
   }
 }
 
+/**
+  * Sums Histograms over time and also computes Max over time of a Max field.
```

Comment:
This is not for histograms.... please make relevant

Reply:
Thanks. Done